config/config.go

```diff
@@ -67,6 +69,11 @@ type ProviderConfig struct {
 	Alias     string
 	Version   string
 	RawConfig *RawConfig
+
+	// Scope records where the Provider was declared in a module tree, so that
+	// it can be copied into child module providers yes still interpolated in
+	// the correct scope.
+	Scope []string
```

Comment:
This was Scope, because I came here to use it for the "interpolation scope". After circling back around, I agree that Path is probably more suitable.